compiler/src/dmd/location.d

```diff
@@ -28,6 +28,7 @@ enum MessageStyle : ubyte
 {
     digitalmars,  /// filename.d(line): message
     gnu,          /// filename.d:line: message, see https://www.gnu.org/prep/standards/html_node/Errors.html
+    sarif         /// JSON SARIF output, see https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html
```

Comment:
You will need to update both `globals.h` and `frontend.h` with this change.

Reply:
Sure, will do.

Reply:
done- https://github.com/dlang/dmd/pull/16986/commits/1205300547f1d478434f00489d0c10d4f5d3ed6f